packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -487,9 +491,28 @@ abstract class ScrollView extends StatelessWidget {
           if (notification.dragDetails != null && focusScope.hasFocus) {
             focusScope.unfocus();
           }
+
           return false;
         },
       );
+    } else if (keyboardDismissBehavior == ScrollViewKeyboardDismissBehavior.interactive) {
+      return Listener(
+        child: scrollableResult,
+        onPointerMove: (PointerMoveEvent details) {
+          double ypos = details.position.dy;
+
+          SystemChannels.textInput.invokeMethod<void>(
+          'TextInput.onPointerMoveForInteractiveKeyboard',
+          <String, dynamic>{'pointerY': ypos});
+        },
+        onPointerUp: (PointerUpEvent details) {
+          double ypos = details.position.dy;
+
+          SystemChannels.textInput.invokeMethod<void>(
+          'TextInput.onPointerUpForInteractiveKeyboard',
+          <String, dynamic>{'pointerY': ypos});
+        }
```

Comment:
Trailing comma here.

Reply:
Don't know why that was missing, will add now!